cs221bot.py

```diff
@@ -191,7 +185,7 @@ async def on_command_error(ctx, error):
         # prints full traceback
         try:
             await ctx.send("```" + "".join(traceback.format_exception(etype, error, trace, 999)) + "```".replace("C:\\Users\\William\\anaconda3\\lib\\site-packages\\", "").replace("D:\\my file of stuff\\cs221bot\\", ""))
-        except Exception:
+        except (discord.HTTPException, discord.Forbidden, discord.InvalidArgument):
```

Comment:
not gonna have this.

only possible exception is the overflow 2000 chars exception.

Reply:
That's fair. I believe that is `discord.InvalidArgument`. I would always recommend having the actual possible exceptions rather than the bare `Exception` which can catch other things that you might want to cause a crash (KeyboardInterrupt, out of memory, etc). In this case, I just went off of what Contex.send can actually throw.